server/const.go

```diff
@@ -1,5 +1,9 @@
 package main
 
 const (
-	PluginName = "mattermost-plugin-metrics"
+	PluginName    = "mattermost-plugin-metrics"
+	tsdbDirName   = "data"
+	metaFileName  = "meta.json"
+	metaVersion1  = 1
+	pluginDataDir = "plugin-data"
```

Comment:
I think this should come from the server. All plugins should use the same dir name for convention.

Reply:
Agreed, ideally yes but do you know whether if there is any convention? I couldn't find. Maybe we can create a ticket and then update later?

Reply:
@lieut-data , @crspeller - Any thoughts on this?

We want to store some plugin specific data in the local filestore.

Reply:
I'm not aware of a plugin directly accessing the filestore like this: traditionally we have only used it indirectly via attachments to posts. I like the idea of a `<pluginDataDir>/<plugin_id>` scoping things.

As an aside, I presume we're architecting this for when the filestore is either `local` or `s3`? Wondering if the latter creates any complexities.

Reply:
Yes, in fact especially for `s3`. The blocks are relatively small ~1MB each. And we only synchronize that in every 2 hours (max block duration).